backend/learnings/views.py

```diff
@@ -234,15 +225,21 @@ def get(self, request, username):
         return Response({"followers": followers, "following": following})
 
 
-class NumberOfWordsLearned(APIView):
+class Dashboard(APIView):
```

Comment:
Rename to something like `QuizResults` or something that will describe what the class does. Since this is not Dashboard specific endpoint.

will also be used in
- profile page
- words learned page

Reply:
And please double check if the profile page is still working

Reply:
Done sir and yes po its still working